agenta-backend/agenta_backend/routers/variants_router.py

```diff
@@ -275,3 +276,22 @@ async def start_variant(
             app_variant_db, envvars, **user_org_data
         )
     return url
+
+
+@router.get("/{variant_id}/logs/")
+async def retrieve_variant_logs(variant_id: str, request: Request):
+    app_variant = await db_manager.get_app_variant_instance_by_id(variant_id)
+    deployment = await db_manager.get_deployment_by_appId(str(app_variant.app.id))
+    if os.environ["FEATURE_FLAG"] == "cloud":
+        try:
+            logs_result = logs_manager.retrieve_cloudwatch_logs(
+                deployment.container_name
+            )
+        except Exception as exc:
+            raise HTTPException(500, {"message": str(exc)})
+        return logs_result
+
+    if os.environ["FEATURE_FLAG"] == "ee":
+        ...
+
```

Comment:
1. let's instead import the right logs manager without the -if elses- here, then simply call it when you need it.

here is a similar example:
https://github.com/Agenta-AI/agenta/blob/main/agenta-backend/agenta_backend/services/app_manager.py#L20-L29
https://github.com/Agenta-AI/agenta/blob/main/agenta-backend/agenta_backend/services/app_manager.py#L89

so that we use the right logs manager for each env and no need for this:
    if os.environ["FEATURE_FLAG"] == "ee":
        ...